homeassistant/const.py

```diff
@@ -81,6 +81,7 @@
 CONF_ENTITY_NAMESPACE = "entity_namespace"
 CONF_ENTITY_PICTURE_TEMPLATE = "entity_picture_template"
 CONF_EVENT = "event"
+CONF_GESTURE = "gesture"
```

Comment:
This is only for deCONZ right now, move to either deconz.const or deconz.deconz_event